api/src/main/java/com/mybooks/api/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package com.mybooks.api.model;
+
+import com.sun.istack.NotNull;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Book {
+    @Id
+    @NotNull
+    @GeneratedValue(generator = "system-uuid")
+    @GenericGenerator(name = "system-uuid", strategy = "uuid")
+    private String id;
+
+    @NotNull
```

Comment:
It'll be nice if you can add `message` , ex : https://www.baeldung.com/spring-validation-message-interpolation